src/modules/status.js

```diff
@@ -0,0 +1,32 @@
+export function saveLocal(list) {
+  window.localStorage.setItem('localTasks', JSON.stringify(list));
+}
+
+export function status(elem, list) {
+  list.forEach((task) => {
+    if (task === elem) {
+      task.isCompleted = !task.isCompleted;
+    }
+  });
+  saveLocal(list);
+}
+
+export function add(list) {
+  list.push({ description: document.querySelector('#newTask').value, isCompleted: false, index: list.length + 1 });
+  document.querySelector('#newTask').value = '';
+  saveLocal(list);
+}
+
+export function updateIndex(list) {
+  let i = 1;
```

Comment:
- It will be better if you use the arrow function instead of the regular function.

Reply:
Thanks, @SaboorSohaib for the review I have implemented the changes.